lxd/.dir-locals.el

```diff
@@ -1,7 +1,7 @@
 ;;; Directory Local Variables
```

Comment:
Hmm, why is this file in there in the first place? :)

Should this be removed from git and a *.el added to gitignore?

Reply:
If you're using emacs, this sets you the right variable to run the tests within emacs. If you don't use it, it won't do anything. I can remove it if you prefer.

Reply:
I'll do a separate commit, but I would live it in the `lxd` dir, because it's only relevant for those unit tests.